portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -195,12 +195,47 @@ definition {
 	}
 
 	@description = "Verify that inputs to the Body Text can be undone and redone"
-	@ignore = "true"
 	@priority = "5"
 	test CanUndoAndRedo {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131245 CanUndoAndRedo pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```

Comment:
You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?